### PR TITLE
containerd: drop unused `fs.RootPath` patch from source control

### PR DESCRIPTION
**Pull request by jpmcb**
**Issue number:**

Closes: n/a

**Description of changes:**

```
This patch was removed from the containerd rpm spec in 7115cde4b81505d87aa26b3ea050711f99e5a31a
and was merged in upstream containerd in https://github.com/containerd/containerd/commit/9f316fa4b5d4089d8ccc75f02e9e348c8b94544c.

This patch removes the unused patch from our source control and doesn't
have any functional change.

Signed-off-by: John McBride <jpmmcb@amazon.com>
```

**Testing done:**

Able to `cargo make` with no problem.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by stmcginnis — APPROVED** [review]
We currently use containerd 1.6.8. Confirmed the change is present in that release:

https://github.com/containerd/containerd/blob/v1.6.8/pkg/cri/opts/container.go#L121

Also verified we are not pulling in that patch in the containerd.spec file, so it looks like it was an oversight that the patch was removed from the package build but the file was left.

**Review by etungsten — APPROVED** [review]
Nice catch. I'll rebase https://github.com/bottlerocket-os/bottlerocket/pull/2588 once this is merged. Or I can cherrypick this commit as part of it. Not sure which is the cleanest, so I'll let you decide!

**Comment by jpmcb** [comment]
@etungsten - I usually like rebase / force pushing, but it's up to you! I'll go ahead and merge this!!

**Pull request merged**
